packages/tsconfig-reference/copy/id/options/emitDecoratorMetadata.md

```diff
@@ -0,0 +1,69 @@
+---
+display: "Menghasilkan Dekorator Metadata"
+oneline: "Menambahkan metadata jenis tambahan ke dekorator dalam kode yang dihasilkan"
+---
+
+Mengaktifkan dukungan eksperimental untuk menghasilkan jenis metadata untuk dekorator yang bekerja dengan modul [`reflect-metadata`](https://www.npmjs.com/package/reflect-metadata).
+
+Contohnya, berikut adalah JavaScript
```

Comment:
Ini konteks bahasa Inggrisnya seperti apakah?

Reply:
`Enables experimental support for emitting type metadata for decorators which works with the module [`reflect-metadata`](https://www.npmjs.com/package/reflect-metadata).

For example, here is the JavaScript`

Reply:
gmn mas, udah sesuai?

Reply:
Gak semua type itu diartikan dengan jenis, di TypeScript Ada banyak kata type yang refer ke tipe data.

Reply:
Berarti type itu dijadiin tipe data ya mas?

Reply:
berarti jadi `menghasilkan tipe data metadata untuk dekorator` gitu ya mas?

Reply:
`For example` itu mungkin terjemahannya `Sebagai contoh`

Reply:
ok mas